.travis.yml

```diff
@@ -19,18 +19,27 @@ matrix:
       - { python: "3.5", env: DJANGO=111 DRF=38 }
       - { python: "3.5", env: DJANGO=20 DRF=38 }
       - { python: "3.5", env: DJANGO=21 DRF=38 }
+      - { python: "3.5", env: DJANGO=21 DRF=39 }
```

Comment:
Think there's more issues with the matrix. tox.ini says there's no django 2.0 (or 2.1) x drf 3.7 env. Can you revisit this?